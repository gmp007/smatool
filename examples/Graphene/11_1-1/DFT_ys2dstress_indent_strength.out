Component: indent_strength
Strain, Stress, Mean Volume, Mean Energy, Mean External Pressure 
0.0, 0.0, 0.0, 0.0 0.0 
0.01, 0.0, 104.69476321, -18.61871997333333, -0.09720000000000001
0.015, 0.0, 104.37761072, -18.609355413333333, -0.144
0.02, 0.0, 104.06023567, -18.59590539, -0.1920666666666667

