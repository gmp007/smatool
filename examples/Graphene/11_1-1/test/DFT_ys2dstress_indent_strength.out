Component: indent_strength
Strain, Stress, Mean Volume, Mean Energy, Mean External Pressure 
0.0, 0.0, 0.0, 0.0 0.0 
0.01, 3.1521631785000004, 106.03168179, -18.604716353333334, 0.07713333333333335
0.015, 4.7771981935, 106.3609282, -18.563898503333334, 0.10693333333333332
0.02, 6.332345683500001, 106.67488658, -18.526140356666666, 0.14013333333333333
0.03, 9.2930120265, 107.25553509000001, -18.39015227333333, 0.18413333333333332
0.04, 12.009607205000002, 107.77081894999999, -18.196608876666666, 0.207
0.05, 14.4596928925, 108.21792978, -17.72010471, 0.2192666666666667
0.06, 16.5989320125, 108.59405923000001, -17.269589516666667, 0.19233333333333333
0.07, 18.376368075, 108.89639891999998, -16.75154978666667, 0.2132666666666667
0.08, 19.720616546000002, 109.1221405, -16.858287540000003, 0.04826666666666667
0.09000000000000001, 20.522673471999997, 109.26847558000001, -16.388970656666668, -0.06746666666666667
0.1, 20.666072473, 109.33259581, -15.87180263, -0.2196

